*** testsBoth/A-13-13B-M-ok.out	2022-11-06 22:59:08.765875000 +0000
--- testsBoth/A-13-13B-M-ok.outhyp	2022-11-10 18:24:16.875521225 +0000
***************
*** 8,13 ****
--- 8,14 ----
  0 - Sair
  Escolha uma opção: 1
  Ficheiro a abrir: gold.dat
+ Abrir: Operação inválida: Problema ao abrir ficheiro: Erro a processar ficheiro gold.dat
  MENU PRINCIPAL
  1 - Abrir
  2 - Guardar
***************
*** 70,76 ****
  0 - Sair
  Escolha uma opção: 10
  Duração da comunicação (em minutos): 40
! Custo da comunicação: 400 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 71,77 ----
  0 - Sair
  Escolha uma opção: 10
  Duração da comunicação (em minutos): 40
! Custo da comunicação: 800 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 108,114 ****
  6 - Mostrar saldo de cliente
  0 - Sair
  Escolha uma opção: 2
! CLIENT|cli001|Manuel Pinheiro|103443|GOLD|YES|4|520|400
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
--- 109,115 ----
  6 - Mostrar saldo de cliente
  0 - Sair
  Escolha uma opção: 2
! CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|0|800
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
***************
*** 165,171 ****
  0 - Sair
  Escolha uma opção: 10
  Duração da comunicação (em minutos): 7
! Custo da comunicação: 140 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 166,172 ----
  0 - Sair
  Escolha uma opção: 10
  Duração da comunicação (em minutos): 7
! Custo da comunicação: 210 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 203,209 ****
  6 - Mostrar saldo de cliente
  0 - Sair
  Escolha uma opção: 2
! CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|520|540
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
--- 204,210 ----
  6 - Mostrar saldo de cliente
  0 - Sair
  Escolha uma opção: 2
! CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|0|1010
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
