*** testsBoth/A-16-03-M-ok.out	2022-10-30 22:57:55.380659000 +0000
--- testsBoth/A-16-03-M-ok.outhyp	2022-11-01 02:20:28.071575446 +0000
***************
*** 42,48 ****
  0 - Sair
  Escolha uma opção: 4
  Número do terminal: 969000
- Adicionar amigo: Operação inválida: O terminal '969000' não existe.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 42,47 ----
***************
*** 68,74 ****
  BASIC|969007|cli003|IDLE|0|0
  BASIC|969008|cli003|SILENCE|0|0
  FANCY|969011|cli001|IDLE|0|0
! FANCY|969013|cli001|SILENCE|0|0|969008
  FANCY|969014|cli002|IDLE|0|0
  FANCY|969015|cli003|IDLE|0|0
  GESTÃO DE TERMINAIS
--- 67,73 ----
  BASIC|969007|cli003|IDLE|0|0
  BASIC|969008|cli003|SILENCE|0|0
  FANCY|969011|cli001|IDLE|0|0
! FANCY|969013|cli001|SILENCE|0|0|969008,969000
  FANCY|969014|cli002|IDLE|0|0
  FANCY|969015|cli003|IDLE|0|0
  GESTÃO DE TERMINAIS
