*** testsBoth/A-19-01-M-ok.out	2022-10-30 22:57:57.557118000 +0000
--- testsBoth/A-19-01-M-ok.outhyp	2022-11-01 02:20:30.575543430 +0000
***************
*** 48,55 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
- 8 - Enviar comunicação de texto
- 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 0
--- 48,53 ----
***************
*** 60,66 ****
  0 - Sair
  Escolha uma opção: 1
  BASIC|969001|cli001|IDLE|0|0
! BASIC|969003|cli001|IDLE|0|0
  BASIC|969006|cli002|IDLE|0|0
  BASIC|969007|cli003|IDLE|0|0
  BASIC|969008|cli003|SILENCE|0|0
--- 58,64 ----
  0 - Sair
  Escolha uma opção: 1
  BASIC|969001|cli001|IDLE|0|0
! BASIC|969003|cli001|OFF|0|0
  BASIC|969006|cli002|IDLE|0|0
  BASIC|969007|cli003|IDLE|0|0
  BASIC|969008|cli003|SILENCE|0|0
