*** testsBoth/A-19-03-M-ok.out	2022-10-30 22:57:57.745500000 +0000
--- testsBoth/A-19-03-M-ok.outhyp	2022-11-01 02:20:30.759541077 +0000
***************
*** 42,48 ****
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 1
- O terminal já estava ligado.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 42,47 ----
***************
*** 76,82 ****
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 1
- O terminal já estava ligado.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 75,80 ----
