*** testsBoth/A-23-06-M-ok.out	2022-11-06 22:59:18.748839000 +0000
--- testsBoth/A-23-06-M-ok.outhyp	2022-11-10 18:24:28.267379739 +0000
***************
*** 202,207 ****
--- 202,208 ----
  0 - Sair
  Escolha uma opção: 6
  Número da comunicação: 2
+ O identificador da comunicação é inválido.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 240,246 ****
  7 - Mostrar terminais com saldo positivo
  0 - Sair
  Escolha uma opção: 5
! CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|3|90|10
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
--- 241,247 ----
  7 - Mostrar terminais com saldo positivo
  0 - Sair
  Escolha uma opção: 5
! CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|3|0|100
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
