*** testsBoth/A-25-03-M-ok.out	2022-11-06 22:59:19.920355000 +0000
--- testsBoth/A-25-03-M-ok.outhyp	2022-11-10 18:24:30.067357054 +0000
***************
*** 85,91 ****
  0 - Sair
  Escolha uma opção: 10
  Duração da comunicação (em minutos): 1
! Custo da comunicação: 20 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 85,91 ----
  0 - Sair
  Escolha uma opção: 10
  Duração da comunicação (em minutos): 1
! Custo da comunicação: 10 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 100,105 ****
--- 100,106 ----
  0 - Sair
  Escolha uma opção: 6
  Número da comunicação: 2
+ O identificador da comunicação é inválido.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
