*** tests/A-08-02-M-ok.out	2022-10-20 15:03:05.337168357 +0100
--- tests/A-08-02-M-ok.outhyp	2022-10-20 16:43:48.739905379 +0100
***************
*** 11,18 ****
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: BASIC|969001|cli002|OFF|0|0
! GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
  3 - Operações de terminal
--- 11,17 ----
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
  3 - Operações de terminal
