*** tests/A-09-01-M-ok.out	2022-10-20 15:03:05.341168404 +0100
--- tests/A-09-01-M-ok.outhyp	2022-10-20 15:45:01.387881608 +0100
***************
*** 11,28 ****
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: Número do terminal: Operações de terminal: Operação inválida: O terminal '969000' não existe.
! GESTÃO DE TERMINAIS
! 1 - Mostrar todos os terminais
! 2 - Registar terminal
! 3 - Operações de terminal
! 0 - Sair
! Escolha uma opção: MENU PRINCIPAL
! 1 - Abrir
! 2 - Guardar
! 3 - Gestão de clientes
! 4 - Gestão de terminais
! 5 - Menu de consultas
! 6 - Ver saldo global
! 0 - Sair
! Escolha uma opção: 
\ Sem newline no fim do ficheiro
--- 11,21 ----
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: Número do terminal: OPERAÇÕES DO TERMINAL
! 1 - Ligar
! 2 - Colocar no silêncio
! 3 - Desligar
! 4 - Adicionar amigo
! 5 - Retirar amigo
! 6 - Pagamento
! 7 - Consultar saldo
