*** tests/A-18-23-M-ok.out	2023-01-26 02:43:14.580083356 +0000
--- tests/A-18-23-M-ok.outhyp	2023-01-26 02:45:38.676901018 +0000
***************
*** 148,153 ****
--- 148,154 ----
  6 - Mostrar saldo de cliente
  0 - Sair
  Escolha uma opção: Identificador do cliente: CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|NO|2|0|0
+ O2I|969013
  GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
